examples/perfect_baroclinic_adjustment_calibration.jl

```diff
@@ -152,7 +152,7 @@ observations = OneDimensionalTimeSeries(data_path, field_names=(:b, :c), normali
 # ### Ensemble model
 
 # First we set up an ensemble model,
-ensemble_size = 10
+ensemble_size = 20
```

Comment:
👀

Reply:
Why not!?

Reply:
well, doesn't it make the problem more expensive? it also converges faster that way (may or may not be realistic...)

Reply:
well, 10 was anyway taken out of a hat... I only made it 20 so that the pdf of the particles at various iterations look a bit nicer :)